repo/packages/C/crate/0/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "crate",
+  "version": "0.1.0-SNAPSHOT",
+  "scm": "https://github.com/crate/crate-mesos-framework.git",
+  "maintainer": "office@crate.io",
+  "description": "A Mesos Framework that allows running and resizing one or multiple Crate database clusters.",
+  "website": "https://crate.io",
+  "tags": ["database", "distributed", "nosql"],
+  "framework": true,
+  "images": {
+    "icon-small": "https://cdn.crate.io/web/2.0/img/crate-avatar_100x100.png"
+  },
+  "postInstallNotes": "Thanks for using Crate! Please read the documentation at https://crate.io/docs/stable/ for usage instructions."
```

Comment:
this file looks good in general.  It would be great to have `preInstallNotes`.  These inform the user of what resource requirements this service needs.